keras_cv/models/object_detection/faster_rcnn.py

```diff
@@ -334,7 +308,7 @@ def __init__(
         )
         self.roi_pooler = _ROIAligner(bounding_box_format="yxyx")
         self.rcnn_head = rcnn_head or RCNNHead(classes)
-        self.backbone = backbone or _resnet50_backbone(include_rescaling)
+        self.backbone = backbone
```

Comment:
To me, this means that `backbone` is required, and we shouldn't have a default value -- right?

Reply:
Maybe let's do `backbone or keras_cv.models.ResNet50(include_rescaling=True)`?

Reply:
that's fine with me

Reply:
(Will also need include_top=False and .as_backbone)